### ISSUE TITLE
Chrome issue when double click

### ISSUE THREAD

**Issue by Seiifer** [likeastore/ngDialog#234]
HI, thanks for your usefull work.

An invisible rectangle upside the dialog taking all the width is selected in blue with Chrome when we double click on the dialog 'opener'.

For example, with Chrome, in your examples (index.html), if we double click on the link such as 'Plain theme' or most of the others too, this will occur.
On firefox, this will select the contenct of the dialog, which is better, but not ideal neither.

Any idea where this comes from and how to prevent it ?
It looks like it's related to the overlay.

Regards

**Comment by egor-smirnov** [comment]
Could you provide example (using service like plunkr) where issue happens?

Without having this it would be hard to find the reason.

**Comment by Seiifer** [comment]
Using Chrome, just run index.html located in the example  folder. Then double click on most of the link opening dialogs.
This doesn't occur when we deactivate the overlay.

**Comment by egor-smirnov** [comment]
This is related to selection of this overlay. The simplest solution is to add 'pointer-events: none' to overlay styles.

I've submitted PR fixing this behaviour.

**Comment by voronianski** [comment]
@egor-smirnov IE support is lacking - http://caniuse.com/#search=pointer-events

**Comment by Seiifer** [comment]
From what I tried, this issue did not occur on IE, neither Firefox, it only occured on Chrome, as far as I know.
However, the double click still trigger the selection of a part of the dialog content.

**Comment by egor-smirnov** [comment]
@voronianski for IE10+ we could use this - https://gist.github.com/egor-smirnov/7b29b7857dc124297b16. For IE10- it's possible to set 'unselectable' attribute, but it have to be done via JS (http://stackoverflow.com/questions/826782/css-rule-to-disable-text-selection-highlighting/4358620#4358620). From my point of view, just adding 'user-select' should be enough. Adding 'unselectable' is maybe over-complication. You thoughts?

@Seiifer I see partial selection issue you mentioned. Going to have a closer look.

**Comment by egor-smirnov** [comment]
As `pointer-events:none` introduces problems in other parts of system (see #266) I decided to revert it back.

Also I think this issue doesn't make a lot of sense as stuff like this could be always tweaked by user of ngDialog. I changed my opinion and now don't want to be any of the elements non-selectable by default. Let the user of the ngDialog decide whether she / he wants non - selectability or not.
